### PR TITLE
New deck lists

### PR DESCRIPTION
**Pull request by SkyJedi**

### CODE REVIEW

File: client/Components/GameBoard/IdentityCard.jsx [hunk 1]
```diff
@@ -7,8 +7,8 @@ import QRCode from 'qrcode';
 import { withTranslation } from 'react-i18next';
 
 class IdentityCard extends React.Component {
-    constructor(props) {
```
Comment (cryogen) [inline client/Components/GameBoard/IdentityCard.jsx]: It is 'proper' to pass the props to the base class, so not sure why you're removing it :)

File: client/Components/GameBoard/IdentityCard.jsx [hunk 2]
```diff
@@ -159,15 +160,29 @@ class IdentityCard extends React.Component {
...
+                ctx.fillText(this.props.t(house).toUpperCase(), houseNames[index].x, houseNames[index].y);
+                ctx.strokeText(this.props.t(house).toUpperCase(), houseNames[index].x, houseNames[index].y);
             });
             this.setState({imageUrl: canvas.toDataURL()});
-
-            this.refs.dummy.parentElement.removeChild(this.refs.dummy);
         });
     }
 
+    findColor(number) {
```
Comment (cryogen) [inline client/Components/GameBoard/IdentityCard.jsx]: ...? :D

File: client/Components/GameBoard/IdentityCard.jsx (hunk 1 above)
Comment (SkyJedi) [reply client/Components/GameBoard/IdentityCard.jsx]: Googles "props base class"
 realization Dawns
I watch the world burn.

File: client/Components/GameBoard/IdentityCard.jsx (hunk 2 above)
Comment (SkyJedi) [reply client/Components/GameBoard/IdentityCard.jsx]: How else will the cards truly discover who they are?

**Pull request merged**